exercises/rotational-cipher/example.rs

```diff
@@ -0,0 +1,16 @@
+pub fn rotate(text: & 'static str, shift_key: u8) -> String {
```

Comment:
I'll look at it and play with it there was a reason I added static I think cause nightly was complaining about something.